liteflow-labs/starter-kit
# Add optional Alchemy provider as fallback when public providers are not responsive

### Description

Add optional Alchemy provider as fallback when public providers are not responsive.
It's based on a new env `NEXT_PUBLIC_ALCHEMY_API_KEY`

### How to test

- With the dev/main branch, open your network console, go to wallet page, notice the error to `https://matic-mumbai.chainstacklabs.com/` and the Mumbai's token with zero of balances
- With this branch, open your network console, go to wallet page, notice the error to `https://matic-mumbai.chainstacklabs.com/`, then see the working request to `https://polygon-mumbai.g.alchemy.com/v2/wtMo0BLwnWWHGWBi-WpVc9d9pFcVPMW3`, Mumbai's token should have the correct balance

### Checklist


### To do before merge

- [x] Add env `NEXT_PUBLIC_ALCHEMY_API_KEY` to uaan
